packages/viewer/src/config/config.ts

```diff
@@ -15,6 +15,7 @@ interface NetworkConfig {
 export interface Config {
   network: NetworkConfig;
   flags: Flags.Flags;
+  streamAttributes: StreamAttributes;
```

Comment:
If possible I think we should limit the number of places this can be set. How about if we just have a watched property `Viewer.streamAttributes`? Whenever it changes, it'll send a request over the WS to update the stream.

Reply:
Makes sense, I'll update 👍